docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -1,161 +1,262 @@
-    pagination.IndicatorSize = new Size(26, 26);
-    pagination.IndicatorSpacing = 8;
-    pagination.Name = "Pagination1";
-    pagination.Size = new Size(300, 50);
-    pagination.BackgroundColor = new Color(1.0f, 1.0f, 1.0f, 0.6f);
-    pagination.IndicatorCount = 5;
-    pagination.SelectedIndex = 1;
-    root.Add(pagination);
+    Pagination pagination = new Pagination();
     ```
 
-Following output is generated when the Pagination is created using property:
-
-![Pagination1](./media/Pagination1.png)
+2. Set the pagination properties:
 
+    ```cs
+   // Path to the images 
+   string _imageUrl = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "images/";
+
+   // Inherited properties of the pagination 
+   _pagination.Name = "Pagination1";
+   _pagination.Size = new Size(500, 150);
+   _pagination.BackgroundColor = new Color(0.4f, 0.56f, 1.0f, 0.7f);
+   _pagination.ParentOrigin = ParentOrigin.Center;
+   _pagination.PositionUsesPivotPoint = true;
+   _pagination.PivotPoint = PivotPoint.BottomCenter;
+
+   // Specific properties of the pagination
+   var _indicatorImageUrlStyle = new PaginationStyle()
+   {
+      IndicatorImageUrl = new Selector<string>
+      {
+         Normal = _imageUrl + "circle_unselected.png",
+         Selected = _imageUrl + "circle_selected.png"
+      }
+   };
+   _pagination.ApplyStyle(_indicatorImageUrlStyle);
+
+   _pagination.IndicatorSize = new Size(40, 40);
+   _pagination.IndicatorSpacing = 60;
+   _pagination.IndicatorCount = 5;
+   _pagination.SelectedIndex = 1;
+   ```
+
+   To set the absolute path of the images that are used, the `Tizen.Applications.Application.Current.DirectoryInfo.Resource` path is used.
+   For more information see
+   [Class Application](https://docs.tizen.org/application/dotnet/api/tizenfx/api/Tizen.Applications.Application.html)
+   and
+   [Class DirectoryInfo](https://docs.tizen.org/application/dotnet/api/tizenfx/api/Tizen.Applications.DirectoryInfo.html).
+
+   You can also set a solid color for the indicators, instead of using images:
+   ```cs
+   _pagination.IndicatorColor = new Color(1.0f, 1.0f, 1.0f, 0.5f);
+   _pagination.SelectedIndicatorColor = Color.Black;
+   ```
+
+3. Add the pagination to the view:
+
+   ```cs
+   _rootView.Add(_pagination);
+   ```
+
+
+The following output is generated when the pagination is created using the property:
+
+| Indicators with images                               | Solid color indicators                               |
+|------------------------------------------------------|------------------------------------------------------|
+|![Pagination1a](./media/PaginationExample_Images.png) | ![Pagination1b](./media/PaginationExample_Color.png) |
+
+
+
+<a name="createWithStyle"></a>
 ## Create with style
 
-To create a Pagination using style, follow these steps:
+To create a pagination using style, follow these steps:
 
-1. Create a style for Pagination:
+1. Create a style for pagination:
```

Comment:
Kindly check for P of pagination.

Reply:
my intention was to use small p everywhere where we talk about an object and P everywhere where I use a class name (3 time in the text).
should I change to? :
"Create an instance of a Pagination class ..."   <- 'class' added

Reply:
Please refer to https://github.com/Samsung/tizen-docs/pull/1366#discussion_r540696858